stories/webapp/accountdashboard/WordPressIntegrationStory.js

```diff
@@ -0,0 +1,105 @@
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { withModalTrigger, Modal, Button, InputWithButton, Icon } from 'lib';
+
+const stories = storiesOf('Web app', module);
+
+const ModalTrigger = withModalTrigger({
+  children: <Icon name="integration" />,
+  types: ['icon-only']
+});
+
+stories.add('WordPressIntegrationStory', () => {
+  const modalSubHeader = (
+    <Fragment>
+      <Icon name="integration" className="m-quarter mr-half" />
+      Wordpress v5.4
+      <span className="text-xs text-green-primary bg-green-90 px-quarter py-tenth inline-flex rounded-sm font-semi-bold mx-quarter">
+        CONNECTED
+      </span>
+      <span className="text-neutral-primary absolute right-0 mr-base">
+        Expires on 30/04/2020
+      </span>
+    </Fragment>
+  );
+
+  return (
+    <ModalTrigger>
+      <Modal.Container>
+        <Modal.Header text={modalSubHeader}>WordPress demo</Modal.Header>
+        <Modal.Body className="has-columns">
+          <div className="modal__column bg-neutral-98">
+            <div className="border border-solid border-neutral-90 rounded bg-white mb-half p-half flex items-center justify-between">
+              <label // eslint-disable-line jsx-a11y/label-has-associated-control
+                htmlFor="preview-url"
+                className="w-1/4 font-semi-bold ml-half"
+              >
+                Preview URL
+              </label>
+              <div className="flex-grow">
+                <InputWithButton
+                  inputId="preview-url"
+                  value="https://cms-wordpress.gatherpreview.com/acme"
+                  onClick={action('copy preview url')}
+                  // onClick={copyElementValueToClipboard}
+                  buttonText="Copy link"
+                  buttonTextAfterClick="Copied!"
+                />
+              </div>
+            </div>
+            <div className="border border-solid border-neutral-90 rounded bg-white mb-half p-half flex items-center justify-between">
+              <label // eslint-disable-line jsx-a11y/label-has-associated-control
```

Comment:
🔴 why do we need ` jsx-a11y/label-has-associated-control` here?

Reply:
because the corresponding id (that the label is for) is on an `InputWithButton`, eslint doesn't understand that the inputId prop is translated down the chain to an `id` on a regular `input`  - related to this comment: https://github.com/gathercontent/gather-ui/pull/695#discussion_r407377805